client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -39,12 +40,14 @@ interface Props {
 	dispute: Dispute;
 	customer: ChargeBillingDetails | null;
 	chargeCreated: number;
+	orderDetails: OrderDetails | null;
 }
```

Comment:
Minor suggestion: explicitly define the props required, in this case we only need the `url`.
```suggestion
	orderDetails: Pick< OrderDetails, 'url' > | null;
```

Reply:
Another option: ask only for the `orderUrl` in the component props.